winbuild/Makefile.vc

```diff
@@ -118,6 +114,10 @@ USE_MBEDTLS = true
 MBEDTLS     = $(WITH_MBEDTLS)
 !ENDIF
 
+!IF ( "$(USE_SSL)"=="true" && "$(USE_WINSSL)"=="true" ) || ( "$(USE_SSL)"=="true" && "$(USE_MBEDTLS)"=="true" ) || ( "$(USE_MBEDTLS)"=="true" && "$(USE_WINSSL)"=="true" )
```

Comment:
I haven't checked the docs whether it is supposed to work with preprocessor directives, but it is supported for commands as documented [here](https://msdn.microsoft.com/en-us/library/2kzfk8c7.aspx?f=255&MSPPError=-2147217396) and I used it in practice without encountering any issues.
Regarding the parantheses you mentioned in the mailing list post: It works, because of a feature called operator precedence which is documented [here](https://msdn.microsoft.com/en-us/library/8t2e8d78.aspx).